server/sample.env

```diff
@@ -1,3 +1,3 @@
 JWT_SECRET=
 MONGO_URI=
-TEAM_NAMES=Bonnie,Sina
\ No newline at end of file
+TEAM_NAMES=Amita
```

Comment:
This code is from the previous PR, when creating branches off of a previous branch please make sure we either adjust the base of our PR or rebase the branch :)

Reply:
Done the changes